### PR TITLE
fix/enhancement(Button) component fix and indentation

### PR DESCRIPTION
**Pull request by gselderslaghs**
## Proposed changes
Fixes a typo and refactored indentation in component css file

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:
- [X] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/main/CONTRIBUTING.md)**.
- [X] My commit messages follows the [conventional commit format](https://github.com/materializecss/materialize/blob/main/CONTRIBUTING.md#submitting-your-pull-request)
- [ ] My change requires a change to the documentation, and updated it accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: sass/components/_buttons.scss [hunk 1]
```diff
@@ -202,21 +206,24 @@
...
 
   &.btn-small {
     --btn-small-height: calc(0.75 * var(--btn-height));
+
     &.halfway-fab {
       bottom: calc(var(--btn-small-height) * -0.5);
     }
-    width: var(--btn-small-height)e;
+
+    width: var(--btn-small-height) e;
```
Comment (wuda-io) [inline sass/components/_buttons.scss:226]: the `e` in the back looks like a typo too.

**Review by wuda-io — APPROVED** [review]

File: sass/components/_buttons.scss (hunk 1 above)
Comment (wuda-io) [reply sass/components/_buttons.scss:226]: can you remove it please, then i will merge it. Thank you.

Comment (gselderslaghs) [reply sass/components/_buttons.scss]: Typo fixed ✓

**Review by wuda-io — APPROVED** [review]
Nice! Lets merge it.

**Pull request merged**